### ISSUE TITLE
Freeze input

### ISSUE THREAD

**Issue by dewiweb** [bitfocus/companion-module-analogway-awj#11]
Hello, 
apparently "freeze input"  functionality is now available on midra4K devices.
example(freeze input 3) from [https://www.analogway.com/files/uploads/produit/download/midra_4k_awj_programmers_guide_v3.2.pdf](https://www.analogway.com/files/uploads/produit/download/midra_4k_awj_programmers_guide_v3.2.pdf):
```
{"op":"replace","path":"DeviceObject/$input/@items/INPUT_3/control/@props/freeze",
"value":true}\0x04
```
Is it possible to add it in this module?

Regards

**Comment by dnmeid** [comment]
Totally possible, but so far this is only announced. At the moment there is no firmware release or simulator with this feature. I guess it will be released in the next few days. It is not a big deal then to add this feature.

**Comment by dewiweb** [comment]
@dnmeid ,
New firmware and simulator released and available  :-)

**Comment by dewiweb** [comment]
@dnmeid Do you have any news about this issue?Is there a new release of awj module in preparation?

**Comment by dnmeid** [comment]
Yes, a module update is in preparation. Also for Compatibility with the upcoming release 4 of WebRCS.

**Comment by dnmeid** [comment]
Functionality is now available in v2.4.0
